### PR TITLE
CAM-11667

### PR DESCRIPTION
**Pull request by AndreyOsipkov**
[CAM-11667](https://app.camunda.com/jira/browse/CAM-11667)

### CODE REVIEW

File: content/reference/rest/task/post-query.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ A JSON object with the following properties:
   <tr>
     <td>processInstanceBusinessKeyIn</td>
     <td>Restrict to tasks that belong to process instances with one of the give business keys.
-        The keys need to be in a comma-separated list.
+       The keys need to be in array.
```
Comment (tmetzke) [inline content/reference/rest/task/post-query.md]:
```suggestion
       The keys need to be in an array.
```

File: content/reference/rest/task/post-query.md [hunk 2]
```diff
@@ -186,7 +186,7 @@ A JSON object with the following properties:
   </tr>
   <tr>
     <td>assigneeIn</td>
-    <td>Only include tasks which are assigned to one of the passed and comma-separated user ids.</td>
+    <td>Only include tasks which are assigned to one of the passed in array user ids</td>
```
Comment (tmetzke) [inline content/reference/rest/task/post-query.md]:
```suggestion
    <td>Only include tasks which are assigned to one of the user ids passed in the array</td>
```

**Comment by AndreyOsipkov** [comment]
corrected

**Comment by tmetzke** [comment]
Perfect, thanks. (y)
Merged it to master.

**Pull request merged**